canonicalwebteam/discourse/parsers/base_parser.py

```diff
@@ -74,11 +74,13 @@ def parse_topic(self, topic):
         soup = self._process_topic_soup(topic_soup)
         self._replace_lightbox(soup)
         sections = self._get_sections(soup)
+        toc = self._generate_toc(soup)
```

Comment:
Can we please call this something less opaque? `contents` maybe? Or something else more descriptive. I believe `table_of_contents` feels a bit long.

If we interrogate this a bit, this actually isn't a table, it's just a data structure containing headings. This module isn't opinionated about how these are used. We could even call it `primary_headings` or something.

Very open to your thoughts, but let's please steer clear of acronyms.

Reply:
Remove this from `BaseParser`